sources/bloodhound/src/bin/kubernetes-checks/checks.rs

```diff
@@ -0,0 +1,714 @@
+use std::{fs::File, path::Path};
+
+use bloodhound::{
+    check_file_not_mode, ensure_file_owner_and_group_root,
+    results::{CheckStatus, Checker, CheckerMetadata, CheckerResult, Mode},
+};
+use serde::Deserialize;
+
+// Bottlerocket doesn't use the standard path for most of these files ¯\_(ツ)_/¯
+const KUBELET_SERVICE_FILE: &str = "/etc/systemd/system/kubelet.service.d/exec-start.conf";
+const KUBELET_KUBECONFIG_FILE: &str = "/etc/kubernetes/kubelet/kubeconfig";
+const KUBELET_CLIENT_CA_FILE: &str = "/etc/kubernetes/pki/ca.crt";
+const KUBELET_CONF_FILE: &str = "/etc/kubernetes/kubelet/config";
+
+// =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<=
+
+pub struct K8S04010100Checker {}
+
+impl Checker for K8S04010100Checker {
+    fn execute(&self) -> CheckerResult {
+        check_file_not_mode(KUBELET_SERVICE_FILE, 0b111111)
+    }
+
+    fn metadata(&self) -> CheckerMetadata {
+        CheckerMetadata {
+            title: "Ensure that the kubelet service file permissions are set to 644 or more restrictive".to_string(),
```

Comment:
The v1.7.0 benchmark seems to use this for all permissions:
```suggestion
            title: "Ensure that the kubelet service file permissions are set to 600 or more restrictive".to_string(),
```

Reply:
This was changed in the 1.7.1 benchmark.